### PR TITLE
Added Normalizr to DRF response

### PR DESCRIPTION
**Pull request by amandasavluchinske**

### CODE REVIEW

File: assets/js/app/BattleDetails/components/ShowBattle.js [hunk 1]
```diff
@@ -3,81 +3,69 @@ import PropTypes from 'prop-types';
...
-            <div className="grid--main-column-third center table--battles">
-              <h2>Challenged: {battle.challenged.email}</h2>
-              <Team
-                winner={battle.winner}
-                pokemons={battle.teams[1].pokemons}
-                trainer={battle.challenged}
-                user={user}
-                teamDescription={'challenged'}
-              />
+      {
```
Comment (victorfsf) [inline assets/js/app/BattleDetails/components/ShowBattle.js]: Why use brackets here?

File: assets/js/app/BattleDetails/components/ShowBattle.js [hunk 2]
```diff
@@ -3,81 +3,69 @@ import PropTypes from 'prop-types';
...
+
 export default function ShowBattle(props) {
-  const { battle, user, isLoading } = props;
+  const { battle, user } = props;
+  const {
+    challengerTeam,
+    challengedTeam,
+  } = filterTeams(battle.challenger, battle.teams);
   return (
     <div>
```
Comment (victorfsf) [inline assets/js/app/BattleDetails/components/ShowBattle.js]: This extra div seem unnecessary.

**Comment by victorfsf** [comment]
Don't forget to fix your PropTypes, eventually. I'm seems a lot of empty `shape()`'s and `arrayOf()`'s

File: assets/js/app/BattleOngoing/components/BattleList.js [hunk 3]
```diff
@@ -13,29 +13,32 @@ export default function BattleList(props) {
...
-                    </table>
-                  </div>),
-            )
+            : list.map((battleId) => {
+              const battle = battles[battleId];
+              const challenger = users[battle.challenger];
+              const challenged = users[battle.challenged];
+              if (battle.winner) return <div />;
+              return (
+                <div>
```
Comment (victorfsf) [inline assets/js/app/BattleOngoing/components/BattleList.js]: This div seems unnecessary (unless the `<table />` tag breaks if you don't use it).

File: assets/js/app/BattleSettled/components/BattleList.js [hunk 4]
```diff
@@ -13,51 +13,54 @@ export default function BattleList(props) {
...
-              ),
-            )
+            : list.map((battleId) => {
+              const battle = battles[battleId];
+              const challenger = users[battle.challenger];
+              const challenged = users[battle.challenged];
+              const winner = users[battle.winner];
+              if (!battle.winner) return <div />;
+              return (
+                <div>
```
Comment (victorfsf) [inline assets/js/app/BattleSettled/components/BattleList.js]: https://github.com/amandasavluchinske/pokebattle/pull/28/files#r211923898

File: assets/js/app/BattleDetails/components/ShowBattle.js (hunk 1 above)
Comment (amandasavluchinske) [reply assets/js/app/BattleDetails/components/ShowBattle.js]: fixed, it was by distraction

File: assets/js/app/BattleDetails/components/ShowBattle.js (hunk 2 above)
Comment (amandasavluchinske) [reply assets/js/app/BattleDetails/components/ShowBattle.js]: agreed kkkkkk

File: assets/js/app/BattleOngoing/components/BattleList.js (hunk 3 above)
Comment (amandasavluchinske) [reply assets/js/app/BattleOngoing/components/BattleList.js]: removed

File: assets/js/app/BattleSettled/components/BattleList.js (hunk 4 above)
Comment (amandasavluchinske) [reply assets/js/app/BattleSettled/components/BattleList.js]: removed

**Comment by amandasavluchinske** [comment]
@victorfsf take a look at it now pls :) I'll do the PropTypes correction in the newest PR once I finish correcting the old ones

**Pull request merged**